### DISCUSSION TITLE
How to Resize and Rotate Grouped Elements

### DISCUSSION THREAD

**Discussion by VADDIJASWANT** [clientIO/joint#2469] [Q&A] [answered]
### Introduction

Hello everyone,

I am working with grouped elements in my project and I am facing a issue with resizing and rotating the grouped elements at the same time, I have followed the [CodePen](https://codepen.io/jointjs/pen/yLvaVrJ?editors=0010) example to create the grouped elements

To reproduce the issue, you can use the same [CodePen](https://codepen.io/jointjs/pen/yLvaVrJ?editors=0010) example. Just remove handles: [] from the object which we are using to create a new ui.selection. After doing this, you can try to resize and rotate the grouped elements in the codepen.

If anyone has encountered this issue before or knows a way to solve it, your input would be greatly appreciated. Thank you!

### Steps to reproduce

1. Open the provided [CodePen](https://codepen.io/jointjs/pen/yLvaVrJ?editors=0010) .
2. Locate the section where handles: [] is used to create a new ui.selection.
3. Remove the code line handles: [] from the object.
4. Attempt to resize and rotate the grouped elements using the selection tools.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
It's currently not supported.

You can see this how the `ui.Selection` can be used to resize the grouped elements: https://codepen.io/jointjs/pen/xxQMopw/a5d4b3f532c574a08583fd09bba4fae8

The rotation does not work as expected there either. The problem is that the `ui.Selection` frame never rotates.
